lib/tasks/scripts/get_bsnl_account_balance.rb

```diff
@@ -0,0 +1,27 @@
+class GetBsnlAccountBalance
+  # (20_000 * 3 patients from current experiments + 15_000 patients from stale experiments) * 4 segments per message
+  # We're unlikely to actually hit this number on any given day, but we'd like to alert early if we're close to the limit
+  MAX_DAILY_MESSAGE_SEGMENT_COUNT = 75_000 * 4
+  BALANCE_EXPIRY_ALERT_DAYS = 3
+
+  class BalanceError < StandardError
```

Comment:
Let's namespace this under `Messaging::Bsnl::`?

Reply:
Cool, wasn't sure if this is a `Messaging` error necessarily, but it should be okay.